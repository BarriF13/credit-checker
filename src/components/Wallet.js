import React, { Component } from 'react'

export class Wallet extends Component {
  render() {
    return (
      <div>
        <h3>wallet balance</h3>
      </div>
    )
  }
}

export default Wallet;
