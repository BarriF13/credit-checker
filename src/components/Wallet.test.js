import { configure } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';

configure({ adapter: new Adapter() });

import React, { Component } from 'react';

import  { shallow } from 'enzyme';

import Wallet from './Wallet';

describe('Wallet', ()=>{
  const wallet = shallow(<Wallet/>);

  it('renders porperly', () =>{
    expect(wallet).toMatchSnapshot();
  });
});
