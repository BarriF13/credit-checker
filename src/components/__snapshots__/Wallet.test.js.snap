// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Wallet renders porperly 1`] = `ShallowWrapper {}`;
